docs/custom-apps.md

```diff
@@ -307,8 +307,6 @@ We want to make it easier to deploy **Zulip Readers** on
 Zulip hardware.  The following document talks about how we want to enable this
 from a code structuring standpoint:
 
-[Writing contrib bots](https://github.com/zulip/zulip/blob/master/contrib_bots/bots/readme.md)
-
```

Comment:
Update: @showell just told me about the history of this doc, so both adding a `deprecated` on top of it or entirely removing it are options.